src/components/ReadMore.tsx

```diff
@@ -0,0 +1,26 @@
+import { useState } from 'react';
+import { ReactNode } from 'react';
+
+import '../styles/ReadMore.scss';
+
+interface IProps {
+  children: ReactNode;
+}
+
+const ReadMore = ({ children }: IProps) => {
+  const text: any = children;
+  const [isReadMore, setIsReadMore] = useState(true);
+  const toggleReadMore = () => {
+    setIsReadMore(!isReadMore);
+  };
+  return !text ? null : text.length < 250 ? text : (
```

Comment:
maybe we can separate this check 
if(!text) {
return null;
}

return text.length < 250 ? text : ( ....

Reply:
fixed

Reply:
Better not to answer before actual code push